docs/getting-started/dockstore-workflows.rst

```diff
@@ -114,31 +114,25 @@ This is the newest way of getting content onto Dockstore and is by far the most
 GitHub Apps, Dockstore can react to changes on GitHub as they are made, keeping Dockstore synced
 with GitHub automatically.
 
-To install our GitHub App on either a repository or organization,
-navigate to the ``/my-workflows`` page and click add workflow. Follow the steps for GitHub Apps and
-you will be redirected to GitHub where you can select which repositories to install the Dockstore
-GitHub app on. The process is shown in the following images.
+Installing the GitHub App is simply -- navigate to ``/my-workflows``, click the ``+`` button on the left hand sidebar, select ``Register using GitHub Apps``, and then click
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can grant the app access to specific repositories within whatever organization you are installing into.
 
-.. figure:: /assets/images/docs/gh-app-reg-1.png
-   :alt: GitHub App Registration on Dockstore
+.. image:: /assets/images/docs/add-workflow-button.png
+   :width: 40 %
 
-   Select Register Using GitHub Apps to get a link to our installation page
+.. image:: /assets/images/docs/register-workflow-github-apps.png
+   :width: 40 %
 
-.. figure:: /assets/images/docs/gh-app-reg-2.png
-   :alt: GitHub App Organizations Page
-
-   Select an organization
-
-.. figure:: /assets/images/docs/gh-app-reg-3.png
-   :alt: GitHub App Organization Install Page
+.. image:: /assets/images/docs/gh-app-reg-1.png
+   :width: 40 %
 
    Install our GitHub App on either all repositories in an organization or on specific repositories
 
 Once you've installed our GitHub app on a repository or organization, you'll need to add a ``/.dockstore.yml`` file to
 the root directory of a branch of the repository that contains your workflow. This file contains information like
 workflow path, test parameter file, workflow name, etc. When a push is made on GitHub to a branch
 with a ``/.dockstore.yml``, Dockstore will add that branch to the corresponding workflow on Dockstore. If the
-workflow doesn't already exist on Dockstore, one will be created.
+workflow doesn't already exist on Dockstore, one will be created. Note that a single ``/.dockstore.yml`` file can describe multiple workflows, if all of those workflows are in the same repository. This will create multiple workflow entries on Dockstore from one ``/.dockstore.yml`` file, one entry for each workflow.
```

Comment:
I don't like "entry" here, especially "one entry for each workflow". A workflow is an entry; this makes it sound like there's an entry and a workflow. I may be missing something though.

Reply:
I'm using "entry" to mean a single entry on Dockstore, which consists of a workflow/tool (not a tool for .dockstore.yml registration of course), its sample JSON if any, etc. If you have 10 workflows in a repo, but only reference one in your .dockstore.yml, the other nine workflows will not become entries.

Reply:
A workflow or tool in Dockstore already consists of its descriptors, JSON, etc.

We use the term "entry" to mean either a Dockstore Tool or a Dockstore workflow. If you go to the logged-in home page, the sidebar on the left shows "My Entries", and it will show a mixture of tools and workflows.

"One entry for each workflow" doesn't sound right to me.

Reply:
I believe we are talking about the same thing. The only distinction I'm making is when I say 
> If you have 10 workflows in a repo, but only reference one in your .dockstore.yml, the other nine workflows will not become entries.

I mean workflow as in a WDL workflow, or a CWL workflow, etc -- which may not necessarily be on Dockstore.

Reply:
I still don't like "entry for workflow". I know "workflow" is an overloaded word, but this is the Dockstore documentation, and a workflow in Dockstore means "descriptor files, JSON", etc.. How about something like:

```
Note that a single /.dockstore.yml file can describe multiple workflows in a single repository.
```